src/main/java/org/example/P29_squareOfANumber.java

```diff
@@ -0,0 +1,14 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class P29_squareOfANumber {
```

Comment:
Use UpperCamelCase formatting for naming classes.

Reply:
Done